README.md

```diff
@@ -35,6 +35,22 @@ Paste that in a Linux shell prompt or in a macOS Terminal.
 
 Paste that in a Powershell (run as Administartor).
 
+### Install an old version 
+
+#### Install on Linux / macOS 
+`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/tipi-build/cli/master/install/install_for_macos_linux.sh > install_tipi.sh)" && /bin/bash -c "chmod +x install_tipi.sh && ./install.sh vx.x.x && rm install_tipi.sh"`
```

Comment:
Scripts like this should not write themselves to the disk, they are supposed to be piped to the bash tool and that's it.

I think better would be : `export TIPI_INSTALL_VERSION="vX.X.X" /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/tipi-build/cli/master/install/install_for_macos_linux.sh ...`